src/main/resources/checkstyle.xml

```diff
@@ -95,12 +95,30 @@
         <property name="message" value="Use JUnit v4 instead."/>
     </module>
 
-    <!-- Forbid shaded packages -->
+    <!-- Always throw propagate result -->
     <module name="RegexpSingleline">
         <property name="format" value="^[ ]*Throwables.propagate\(.*"/>
         <property name="message" value="Always throw the result of Throwables.propagate."/>
     </module>
 
+    <!-- Checks that LogValueBuilder without reference is not used -->
```

Comment:
I'm not crazy about these being regex matches.  Seems pretty fragile.  But I don't have a better solution.